Gems/EMotionFX/Code/EMotionFX/Pipeline/RCExt/Actor/ActorBuilder.cpp

```diff
@@ -366,7 +366,45 @@ namespace EMotionFX
                 auto mesh = azrtti_cast<const SceneDataTypes::IMeshData*>(it->second);
                 if (mesh)
                 {
-                    outMeshIndices.push_back(nodeIndex);
+                    outMeshIndices.emplace_back(nodeIndex);
+
+                    // Don't need to add mesh node except it is a parent of other joint / mesh node.
```

Comment:
nit: it is -> if it is
nit: other -> another

Update: _Don't need to add a mesh node except if it is a parent of another joint / mesh node._

Reply:
fixed